repos/system_upgrade/el7toel8/actors/firewalldfactsactor/actor.py

```diff
@@ -0,0 +1,56 @@
+from leapp.actors import Actor
+from leapp.models import FirewalldFacts
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+from leapp.libraries.actor import private
+
+import os
+import xml.etree.ElementTree as ElementTree
+
+
+class FirewalldFactsActor(Actor):
+    """
+    Provides data about firewalld
```

Comment:
Use imperative style for docstring.

Reply:
I'm not sure what you're asking for. I copied the style from `actors/systemfacts/actor.py`. Do you have a suggested docstring?